### ISSUE TITLE
Add theme to desktop conductor UI

### ISSUE THREAD

**Issue by Yugue** [flutter/flutter#90982]
The current desktop conductor UI uses the default Flutter theme. A global theme should be added to `main.dart`. A single PR should be made to address this.

**Comment by gspencergoog** [comment]
Sorry, @Yugue can you clarify what this issue as asking for, and how does it relate Desktop support for the Framework (the `a: desktop` label)?

**Comment by gspencergoog** [comment]
Actually, this doesn't look like a Desktop framework related issue (Desktop Triage), so I'm going to just remove the label.

**Comment by Yugue** [comment]
@gspencergoog I am using that label because the app I am working on is a desktop app of the Flutter SDK release tool. Isn't this what the label is used for?

**Comment by gspencergoog** [comment]
@Yugue No, it is used to indicate an issue having to do with Flutter's desktop support. The label description is a bit vague on that point, though, I realize.

**Comment by Yugue** [comment]
@gspencergoog ok sounds good.
